core-library/src/test/java/org/silverpeas/core/notification/user/UserSubscriptionNotificationSendingHandlerTest.java

```diff
@@ -52,8 +52,8 @@ class UserSubscriptionNotificationSendingHandlerTest {
   @BeforeEach
   public void setup() {
     request = mock(HttpServletRequest.class);
-    CacheServiceProvider.getThreadCacheService().clearAllCaches();
-    CacheServiceProvider.getRequestCacheService().clearAllCaches();
+    CacheAccessorProvider.getThreadCacheAccessor().getCache().clear();
+    CacheAccessorProvider.getThreadCacheAccessor().getCache().clear();
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏ duplicate of above line